taxi/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
+from django.http import HttpRequest, HttpResponse
+from .models import Driver, Manufacturer, Car
```

Comment:
fix the imports. python modules first, then installed modules, then your own and each group should be separated by blank line